src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -687,6 +687,8 @@ void StorageReplicatedMergeTree::createNewZooKeeperNodes()
         futures.push_back(zookeeper->asyncTryCreateNoThrow(zookeeper_path + "/zero_copy_s3/shared", String(), zkutil::CreateMode::Persistent));
         futures.push_back(zookeeper->asyncTryCreateNoThrow(zookeeper_path + "/zero_copy_hdfs", String(), zkutil::CreateMode::Persistent));
         futures.push_back(zookeeper->asyncTryCreateNoThrow(zookeeper_path + "/zero_copy_hdfs/shared", String(), zkutil::CreateMode::Persistent));
+        futures.push_back(zookeeper->asyncTryCreateNoThrow(zookeeper_path + "/zero_copy_cfs", String(), zkutil::CreateMode::Persistent));
```

Comment:
A question - why do we use different names of the nodes for different types of disks?